src/components/section/index.js

```diff
@@ -57,10 +60,12 @@ Section.propTypes = {
     PropTypes.node,
   ]).isRequired,
   style: PropTypes.shape({}),
+  stretch: PropTypes.bool,
```

Comment:
It might be a good idea to add warning when setting stretch prop on a grid that is not stretched. This case would do nothing and caller wouldn't be notified in any way why it is not working.

Reply:
This is fixed.